src/files/files.service.ts

```
@@ -0,0 +1,66 @@
+import {
+  GetObjectCommand,
+  PutObjectCommand,
+  S3Client,
+} from '@aws-sdk/client-s3';
+import { getSignedUrl } from '@aws-sdk/s3-request-presigner';
+import { Injectable, InternalServerErrorException } from '@nestjs/common';
```

ESLint

[error] 'InternalServerErrorException' is defined but never used.

(@typescript-eslint/no-unused-vars)